### PR TITLE
Add issue and PR template from CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by rreusser**
I've copied with very little modification the guidelines from CONTRIBUTING.md. The goal is to elevate the guidelines to a higher level of visibility so that people are more effectively steered toward the forum or good practices. Hopefully will save some repetitive back and forth for @etpinard.

Modify as needed. 👍

### CODE REVIEW

File: .github/PR_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Thanks for your interest in plotly.js!
```
Comment (etpinard) [inline .github/PR_TEMPLATE.md]: Oh. I think this file needs to be named `PULL_REQUEST_TEMPLATE.md` according to https://github.com/blog/2111-issue-and-pull-request-templates

Comment (rreusser) [inline .github/PR_TEMPLATE.md]: Oops. My mistake. Got in a hurry. Fixed.

**Comment by etpinard** [comment]
:dancer: merge away test have passed.

**Comment by rreusser** [comment]
I'm seeing one failed, but looks like the intermittent double-click thing.

**Pull request merged**